.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+/.bundle/ @cockroachdb/docs-infra-prs
+/.github/ @cockroachdb/docs-infra-prs
+/__tests__/ @cockroachdb/docs-infra-prs
+/_layouts/  @cockroachdb/docs-infra-prs
+/_plugins/  @cockroachdb/docs-infra-prs
+/archived/  @cockroachdb/docs-infra-prs
+/css/ @cockroachdb/docs-infra-prs
+/fonts/ @cockroachdb/docs-infra-prs
+/githooks/  @cockroachdb/docs-infra-prs
+/jekyll-algolia-dev/  @cockroachdb/docs-infra-prs
+/js/  @cockroachdb/docs-infra-prs
+/netlify/ @cockroachdb/docs-infra-prs
+/vale/  @cockroachdb/docs-infra-prs
+_redirects  @cockroachdb/docs-infra-prs
+.gitattributes  @cockroachdb/docs-infra-prs
+.gitignore  @cockroachdb/docs-infra-prs
+.htmltest.yaml  @cockroachdb/docs-infra-prs
+.vale.ini @cockroachdb/docs-infra-prs
+Gemfile @cockroachdb/docs-infra-prs
+LICENSE @cockroachdb/docs-infra-prs
+Makefile  @cockroachdb/docs-infra-prs
+navgoco_license @cockroachdb/docs-infra-prs
```

Comment:
I don't know if this should be on `docs-infra-prs`. What are your thoughts? Also, would you agree that it's a problem that it's hard for people to tell what the license applies to? Do you know if there is any context we can add at the top of the content or file name without 'adjusting' the license?

Reply:
Upon first glance, I can imagine people will have no idea what `navgoco_license` means without having to look it up. I know it's the software we use to render the sidebar because I've had to research it.

I'm not sure if the license is even necessary for us to include in the repo, because the project has been defunct for 7 years. We can discuss that next Thursday.